### DISCUSSION TITLE
Unable to create PR due to "Validation failed: must be a collaborator"

### DISCUSSION THREAD

**Discussion by mausam-patel** [microsoft/winget-pkgs#141390] [General]
I am trying to open a PR to merge a branch from a forked repo into this repo. An example PR of this previously with the same source/target branches is 
https://github.com/microsoft/winget-pkgs/pull/86812. However, I am receiving the following error: 

`Pull request creation failed. Validation failed: must be a collaborator`

Have there been any changes to the contributions policy or permissions? Any insight/assistance would be helpful!

**Comment by Hueristic** [comment]
All part of their "open" sauce takeover.
